CMakeLists.txt

```diff
@@ -134,6 +134,16 @@ include(cmake/SetupCMakeBasics.cmake)
 include(cmake/SetupCompilerFlags.cmake)
 include(cmake/SetupUmpireThirdParty.cmake)
 
+if (WIN32)
+  if ((NOT UMPIRE_ENABLE_FILESYSTEM) OR (NOT "${CMAKE_CXX_STANDARD}" STREQUAL "17"))
```

Comment:
👍

Reply:
@kab163 this should check that the standard is at least c++17